.github/workflows/nix-workflow.yml

```diff
@@ -0,0 +1,21 @@
+name: "Nix build"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  nix-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [musl, arm64-musl]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v18
+      with:
+        extra_nix_config: |
```

Comment:
This here is to access precompiled packages cache for your overlays?

Reply:
yes, we upload to the cache in the overlays CI